libs/async_cuda/include/hpx/async_cuda/cuda_future.hpp

```diff
@@ -0,0 +1,345 @@
+//  Copyright (c) 2020 John Biddiscombe
+//  Copyright (c) 2016 Thomas Heller
+//  Copyright (c) 2016 Hartmut Kaiser
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <hpx/config.hpp>
+#include <hpx/async_cuda/cuda_event.hpp>
+#include <hpx/async_cuda/cuda_exception.hpp>
+#include <hpx/basic_execution.hpp>
+#include <hpx/concurrency.hpp>
+#include <hpx/debugging/print.hpp>
+#include <hpx/futures/future.hpp>
+#include <hpx/memory.hpp>
+#include <hpx/runtime_local/thread_pool_helpers.hpp>
+#include <hpx/threading_base.hpp>
```

Comment:
These need to be updated (https://cdash.cscs.ch//viewBuildError.php?buildid=110625)
```suggestion
#include <hpx/config.hpp>
#include <hpx/async_cuda/cuda_event.hpp>
#include <hpx/async_cuda/cuda_exception.hpp>
#include <hpx/modules/basic_execution.hpp>
#include <hpx/modules/concurrency.hpp>
#include <hpx/debugging/print.hpp>
#include <hpx/futures/future.hpp>
#include <hpx/modules/memory.hpp>
#include <hpx/runtime_local/thread_pool_helpers.hpp>
#include <hpx/modules/threading_base.hpp>
```